kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -36,7 +36,35 @@ const fetchNodes = ({ dispatch }, id) => {
   });
 };
 
+
+const searchNodes = ({ dispatch }, params, page) => {
+  // Get the collection from ContentNodeResource.
+  const contentCollection = Kolibri.resources.ContentNodeResource.getCollection();
+  const wantedFields = ['pk', 'title', 'kind', 'instance_id', 'content_id', 'description', 'files'];
```

Comment:
they differ by each specific search, how could I know? say a node has thousands of ancestors